ui/tests/acceptance/client-detail-test.js

```diff
@@ -1140,12 +1140,21 @@ module('Acceptance | client detail', function (hooks) {
   testFacet('Status', {
     facet: ClientDetail.facets.status,
     paramName: 'status',
-    expectedOptions: ['Pending', 'Running', 'Complete', 'Failed', 'Lost'],
+    expectedOptions: [
+      'Pending',
+      'Running',
+      'Complete',
+      'Failed',
+      'Lost',
+      'Unknown',
+    ],
     async beforeEach() {
       server.createList('job', 5, { createAllocations: false });
-      ['pending', 'running', 'complete', 'failed', 'lost'].forEach((s) => {
-        server.createList('allocation', 5, { clientStatus: s });
```

Comment:
> issue: I think this will impact the RescheduleEventTimeline in the /allocations/allocation view to display the wrong color.

I might need some help with this one; I can't seem to trigger a situation where RescheduleEventTimeline has a Disconnected event.

Reply:
^--- Disregard the above; I see `Unknown` in the event timeline now